### PR TITLE
Feature/#5 cv page who i am fe

### PR DESCRIPTION
**Pull request by claudianaghi99**
Please review my code.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,12 +6,20 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <script src="https://kit.fontawesome.com/b21073efe8.js" crossorigin="anonymous"></script>
```
Comment (mariangoia91) [inline public/index.html]: you can install fontawesome using npm

`npm install --save @fortawesome/fontawesome-free`

File: public/index.html [hunk 2]
```diff
@@ -6,12 +6,20 @@
...
+
   </head>
   <body>
     <noscript>
       <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
     <div id="app"></div>
     <!-- built files will be auto injected -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"
```
Comment (mariangoia91) [inline public/index.html]: are you sure this is needed if you've already installed bootstrap using npm?

File: src/components/Main.vue [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  padding-right: 80%;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+
+  color: #23262b;
+}
+.about-contact h2{
```
Comment (mariangoia91) [inline src/components/Main.vue]: missing space before bracket

File: src/main.js [hunk 4]
```diff
@@ -1,18 +1,17 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 
-import router from "./router";
-import axios from "axios";
-import VueAxios from "vue-axios";
-import store from "./store";
-import 'bootstrap/dist/css/bootstrap.css'
-import 'bootstrap-vue/dist/bootstrap-vue.css'
+// import router from "./router";
```
Comment (mariangoia91) [inline src/main.js]: please remove commented code

**Review by mariangoia91 — CHANGES_REQUESTED** [review]

**Review by tant-rares-30127 — APPROVED** [review]
She did good.

**Pull request merged**